RMSD.py

```diff
@@ -113,6 +123,11 @@ def RMSD(ref_mol2, query_mol2):
     # set up a log file for RMSD
     logFile = open('%s/%s' % (directory, opt.output) ,'a')
 
+    has_outlier = False
+    if opt.cutoff > 0.0:
```

Comment:
This says ">" but the help statement says "greater than or equal to"

Reply:
Well if you set it to 0.0 you're not going to get any molecules so my thought was a user could set the cutoff to 0.0 if they don't want outlier data reported? I figured there might be molecule sets where we don't want outlier data, so the thought was negative values don't make sense, 0.0 indicates not to create the file, > 0.0 will create a file with a line for any molecule that has RMSD data above that value

Reply:
oh, I see that now. for some reason I thought the default option was to not check for outliers. this approach should be fine. if store_outliers & 2.0 cutoff are the default settings, we probably need to document that this was an arbitrary cutoff....